packages/runtime-html/src/templating/controller.ts

```diff
@@ -762,11 +773,15 @@ export class Controller<C extends IViewModel = IViewModel> implements IControlle
     initiator: IHydratedController,
     _parent: IHydratedController | null,
   ): void | Promise<void> {
+    let prevActivation: void | Promise<void> = void 0;
     switch ((this.state & ~State.released)) {
       case State.activated:
-        // We're fully activated, so proceed with normal deactivation.
         this.state = State.deactivating;
         break;
+      case State.activating:
+        this.state = State.deactivating;
+        prevActivation = this.$promise?.catch(noop); /** justification: because we are about to deactivate, the error from activation can be ignored */
```

Comment:
I think we can afford to add quite a bit of helping text in a dev build around here.
```suggestion
        // we are about to deactivate, the error from activation can be ignored
        prevActivation = this.$promise?.catch(__DEV__
          ? ex => {
            // ...
          }
          : noop
        )
```

Reply:
Good idea! Added some logging.